### DISCUSSION TITLE
Gravatar for AI agents (labeled! transparent!)

### DISCUSSION THREAD

**Discussion by digitalwaveride** [Automattic/gravatar#51] [Ideas]
AI agents - as long as it is very transparent, and very obviously labeled as bot, could there be a new type of Gravatar user? 

Could there be via the API a way to create such profiles, that then clearly will be marked as an agent, via the avatar, hovercard, and profile page?

---
Source of request: User Interview p8Slzc-3hc-p2
(This request was not made by the OP)

**Comment by devmangel** [comment]
Hello @digitalwaveride 

This is an interesting proposal to create a distinct category of Gravatars for AI agents. It would indeed be beneficial to ensure that these profiles are transparently identified as bots, maintaining clarity and trust in digital interactions.

To implement this, the API could be extended to allow registration of AI agents as a separate entity. This would involve:

API Endpoint for AI Agent Registration: Develop a new endpoint where developers can register their AI agents, specifying mandatory fields that make it clear the entity is a bot.
Custom Avatar and Hovercard: Introduce customizable avatars and hovercards that are distinctly different, maybe even incorporating a standard bot icon or watermark to reinforce that these are not human-operated profiles.
Profile Page Customization: Enable special layouts or themes for the profile pages of AI agents, which would include clear markings as you suggested.
The key would be to enforce strict guidelines on how these avatars are used to prevent any misleading representations. Perhaps a verification process could be established to ensure that only legitimate bots are registered under this new type.

Would love to hear more thoughts on how we might refine or expand on these ideas!

**Comment by Qmig** [comment]
Love this
